### PR TITLE
test: refactor test utils

### PR DESCRIPTION
**Pull request by anshumanv**
**What kind of change does this PR introduce?**

refactor 

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
- refactor test utils
- 
**Does this PR introduce a breaking change?**
No

**Other information**
WIP

### CODE REVIEW

File: test/build/entry/scss/scss.test.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
 /* eslint-disable   node/no-unpublished-require */
-const { run, runInstall } = require('../../../utils/test-utils');
+const { run } = require('../../../utils/test-utils');
 
 describe('entry point', () => {
     it('should support SCSS files', async () => {
-        await runInstall(__dirname);
```
Comment (alexander-akait) [inline test/build/entry/scss/scss.test.js:6]: Why we remove this?

Comment (anshumanv) [reply test/build/entry/scss/scss.test.js:6]: we do it in `prepareSuite.js` this is redundant

File: test/init/init.test.js [hunk 2]
```diff
@@ -26,8 +26,8 @@ const readFromPkgJSON = (path) => {
 const readFromWebpackConfig = (path) => readFileSync(join(path, 'webpack.config.js'), 'utf8');
 
 describe('init command', () => {
-    beforeAll(async () => {
-        await mkdir(rootAssetsPath);
+    beforeAll(() => {
+        if (!existsSync(rootAssetsPath)) mkdirSync(rootAssetsPath);
```
Comment (alexander-akait) [inline test/init/init.test.js:30]: Please prefer to use `if () {}`, bracers is good practice, we need improve eslint config for all webpack repos and using them across all repos

Comment (anshumanv) [reply test/init/init.test.js:30]: yep on it

Comment (anshumanv) [reply test/init/init.test.js]: done

**Review by alexander-akait — APPROVED** [review]
Let's merge after https://github.com/webpack/webpack-cli/pull/2573, to avoid conflicts, a lot of changes

**Review by alexander-akait — DISMISSED** [review]
Feel free to merge

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Pull request merged**